java/src/jmri/jmrit/ctc/CTCMain.java

```diff
@@ -1,6 +1,8 @@
 /**
  *  @author Gregory J. Bedlek Copyright (C) 2018, 2019
  * 
+ * Release notes(files): /JMRI/help/en/releasenotes/current-draft-note.shtml Section: CTC Tool section.
```

Comment:
I'm not sure that it's good to add reference to the release notes here. That release note is only valid for the next major version and I guess it will then be removed from the release note.

Reply:
Daniel:
(P.S. I sent this originally as part of a normal email, you may have gotten this already):

Thank you for the suggestion.  But the reason I put this comment line in there is so that I as a programmer can remember where in the huge directory structure the file is located, so that I can edit it.  It is only for "me".  At least if the name changes, the directory will probably still be the same.  So at least I have an idea where to look.  I'm not too good at remembering where 1000's of things are.  I'm only a lowly programmer shmuck! :-) :-)

Now that you understand why the comment is there (for my memory, nothing else), if you still think I should remove it, I will.  I tell you all this because maybe I don't understand your comment (because of my my lack of understanding JMRI in general).

And I really want to hear your opinion as to whether that is a good place for the memory aid for me.  I tend to put those reminders right in the source.  Maybe I shouldn't?  And I'm asking your opinion on that too please, not just this one.

Sincerely

Greg

Reply:
Daniel:
I talked at GREAT length to Dave Sand, and he explained to me why you said what you said.  I apologize.  I now realize that others will be looking at this, and they will think that it is something important, and not just a reminder for me.  I WILL remove it in my next push.  Sorry.

Reply:
Gregory:
Thanks! We are here to help you. Not long ago, I too was new to JMRI development so I know the difficulty with learning where everything is located in the JMRI project. There are still a lot of blind spots of JMRI development that I don't understand.

My suggestion is that you have your own document with useful notes, and that you keep that file outside the JMRI project. I have many such documents for things I'm working on.

Did you sent me an email? If so, to which email address? I haven't got any private email from you so my question is if there is a problem with my email? My email is `db123 AT bergqvist.se`, replace AT with @ . Feel free to email me if you want.

Reply:
Daniel:
Absolutely what I've done!  Thanks for your help in explaining further.

That comment is gone, and actually that "path" is in the JMRI documentation.

Web page: https://www.jmri.org/help/en/html/doc/Technical/gitdeveloper.shtml

Under "Develop and Test Locally", bottom paragraph.

That is good enough for me!

Sincerely

Greg

Reply:
Don't worry about the email.  I realize now that I MUST reply within GitHub otherwise the email system eats it up I think.